test failure Ubuntu TestARMLEConstant.test_dynamic_predict

Ubuntu pythonxy  testing shows this error (seen it before)

https://launchpadlibrarian.net/143376706/buildlog_ubuntu-raring-amd64.statsmodels_0.5.0~ppa17~revno-1419~raring1_UPLOADING.txt.gz

precision issue or serious?

```
======================================================================
FAIL: statsmodels.tsa.tests.test_ar.TestARMLEConstant.test_dynamic_predict
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/buildd/statsmodels-0.5.0~ppa17~revno/debian/python-statsmodels/usr/lib/python2.7/dist-packages/statsmodels/tsa/tests/test_ar.py", line 159, in test_dynamic_predict
    assert_almost_equal(fv, res2.fcdyn[start:end+1], DECIMAL_4)
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 452, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 812, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 645, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 20.9302325581%)
 x: array([ 11.23050331,  13.2373889 ,  19.6641709 ,  27.00383557,
        33.57868933,  44.90793538,  47.95874902,  45.09354677,
        36.50663235,  28.7882511 ,  23.48720679,  22.53217807,...
 y: array([ 11.23047,  13.23731,  19.66407,  27.00373,  33.57858,  44.90784,
        47.95866,  45.09347,  36.50656,  28.78818,  23.48711,  22.53206,
        25.68737,  31.07822,  38.54836,  45.41845,  49.67139,  49.37283,...

----------------------------------------------------------------------
Ran 2745 tests in 725.733s

FAILED (SKIP=14, failures=1)
```

--- a/statsmodels/tsa/tests/test_ar.py
+++ b/statsmodels/tsa/tests/test_ar.py
@@ -114,6 +114,8 @@ class TestAROLSNoConstant(CheckARMixin):
                 self.res2.FVOLSn15start312, DECIMAL_4)
 
         #class TestARMLEConstant(CheckAR):
+
+
 class TestARMLEConstant(object):
     @classmethod
     def setupClass(cls):
@@ -123,7 +125,12 @@ class TestARMLEConstant(object):
 
     def test_predict(self):
         model = self.res1.model
-        params = self.res1.params
+        # for some reason convergence is off in 1 out of 10 runs on
+        # some platforms. i've never been able to replicate. see #910
+        params = np.array([ 5.66817602,  1.16071069, -0.39538222,
+                           -0.16634055,  0.15044614, -0.09439266,
+                           0.00906289,  0.05205291, -0.08584362,
+                           0.25239198])
         assert_almost_equal(model.predict(params), self.res2.FVMLEdefault,
                 DECIMAL_4)
         assert_almost_equal(model.predict(params, start=9, end=308),
@@ -149,6 +156,12 @@ class TestARMLEConstant(object):
                 self.res2.FVMLEstart2end7, DECIMAL_4)
 
     def test_dynamic_predict(self):
+        # for some reason convergence is off in 1 out of 10 runs on
+        # some platforms. i've never been able to replicate. see #910
+        params = np.array([ 5.66817602,  1.16071069, -0.39538222,
+                           -0.16634055,  0.15044614, -0.09439266,
+                           0.00906289,  0.05205291, -0.08584362,
+                           0.25239198])
         res1 = self.res1
         res2 = self.res2
 
@@ -157,52 +170,52 @@ class TestARMLEConstant(object):
 
         # 9, 51
         start, end = 9, 51
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn[start:end+1], rtol=rtol)
 
         # 9, 308
         start, end = 9, 308
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn[start:end+1], rtol=rtol)
 
         # 9, 333
         start, end = 9, 333
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn[start:end+1], rtol=rtol)
 
         # 100, 151
         start, end = 100, 151
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn2[start:end+1], rtol=rtol)
 
         # 100, 308
         start, end = 100, 308
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn2[start:end+1], rtol=rtol)
 
         # 100, 333
         start, end = 100, 333
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn2[start:end+1], rtol=rtol)
 
         # 308, 308
         start, end = 308, 308
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn3[start:end+1], rtol=rtol)
 
         # 308, 333
         start, end = 308, 333
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn3[start:end+1], rtol=rtol)
 
         # 309, 333
         start, end = 309, 333
-        fv = res1.predict(start, end, dynamic=True)
+        fv = res1.model.predict(params, start, end, dynamic=True)
         assert_allclose(fv, res2.fcdyn4[start:end+1], rtol=rtol)
 
         # None, None
         start, end = None, None
-        fv = res1.predict(dynamic=True)
+        fv = res1.model.predict(params, dynamic=True)
         assert_allclose(fv, res2.fcdyn[9:309], rtol=rtol)
 
 
@@ -220,7 +233,6 @@ class TestAutolagAR(object):
             k_ar = r.k_ar
             k_trend = r.k_trend
             log_sigma2 = np.log(r.sigma2)
-            #import ipdb; ipdb.set_trace()
             aic = r.aic
             aic = (aic - log_sigma2) * (1 + k_ar)/(1 + k_ar + k_trend)
             aic += log_sigma2

